packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -799,22 +800,20 @@ export const MultipleBanners: Story = {
                   </ItinerarySegmentBanner>
                   <Separator />
                   <ItinerarySegmentBanner>
-                    <ItineraryBadgeList>
-                      <Stack spacing="200">
-                        <ItineraryBadgeListItem icon={<Location color="secondary" />}>
-                          You’ll depart from a different place in New York: John F. Kennedy
-                          International.
-                        </ItineraryBadgeListItem>
-                        <ItineraryBadgeListItem icon={<Location color="secondary" />}>
-                          You’ll depart from a different place in New York: John F. Kennedy
-                          International.
-                        </ItineraryBadgeListItem>
-                        <ItineraryBadgeListItem icon={<Accommodation color="secondary" />}>
-                          We won’t cover your overnight stay. Hotel coverage is only available if
-                          the disruption happens during the trip. If you want to avoid extra hotel
-                          costs please choose a different alternative or a refund.
-                        </ItineraryBadgeListItem>
-                      </Stack>
+                    <ItineraryBadgeList spacing={SPACINGS.ONE_HUNDRED}>
```

Comment:
Interesting that it works nice! Could you please add an example also at "Badge List"? With a control maybe. WDYT?

At the same time, I don't know if it makes sense to document this behavior here… I would assume designs are consistent regarding the spacing between items inside an Itinerary 💭 

So I'd leave it up to you, really

Reply:
Added! 👍